TMS.Net07.HomeworkDateToDayOfWeeks/TMS.Net07.Homework2 - middle/Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TMS.Net07.Homework2___middle
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string input;
+            DateTime inputDate, maxRange = DateTime.Parse("31.12.2999");
```

Comment:
String parsing is a rather long operation.. There is more native way to create a DateTime:
https://www.tutorialsteacher.com/csharp/csharp-datetime

Reply:
Done